org.adempiere.base.process/src/org/compiere/process/CreateWindowFromTable.java

```diff
@@ -173,6 +173,7 @@ protected String doIt () throws Exception
 			}
 
 			MTab tab = new MTab(window);
+			tab.setEntityType(entityType);

```

Comment:
Would be better if the process can take an optional entity type parameter. For e.g, I might want to create a new custom window for C_BPartner table and wants to create it with User Defined entity type.

Reply:
It is just a suggestion, I didn't put this as request for changes. You are free to approve and merge this PR.